### PR TITLE
s/thealphanerd/mylesborins

### PR DESCRIPTION
**Pull request by MylesBorins**
new year new look

### CODE REVIEW

**Comment by jbergstroem** [comment]
🙊

**Comment by MylesBorins** [comment]
fixed

File: README.md [hunk 1]
```diff
@@ -200,4 +200,4 @@ Build and test orchestration is performed by [Jenkins][21].
 [20]:   https://tessel.io/
 [21]:   https://jenkins.io/
 [pivotal]: https://www.pivotalagency.com.au/
-[securo]: http://securogroup.com/
\ No newline at end of file
+[securo]: http://securogroup.com/
```
Comment (gibfahn) [inline README.md]: What's with this change? Loss of newline at the end of the file?

Comment (MylesBorins) [inline README.md]: looks like it is adding one. I used the GitHub ui for this, likely added the new line

**Pull request merged**